tests/func/test_repro.py

```diff
@@ -1204,19 +1198,18 @@ def test(self):
             )
             self.assertTrue(run_stage is not None)
 
-        self.assertTrue(os.path.exists(run_output))
+            self.assertTrue(os.path.exists(run_output))
 
-        # Pull
-        self.dvc.remove(import_stage.path, outs_only=True)
-        self.assertFalse(os.path.exists(import_output))
+            # Pull
+            self.dvc.remove(import_stage.path, outs_only=True)
+            self.assertFalse(os.path.exists(import_output))
 
-        shutil.move(self.local_cache, self.external_cache_id)
-        self.assertFalse(os.path.exists(self.local_cache))
+            shutil.move(self.local_cache, cache_id)
+            self.assertFalse(os.path.exists(self.local_cache))
 
-        with StaticFileServer():
             self.dvc.pull(import_stage.path, remote="mycache")
 
-        self.assertTrue(os.path.exists(import_output))
+            self.assertTrue(os.path.exists(import_output))
```

Comment:
We may use normal asserts as long as we use pytest to run the thing.

Reply:
@Suor Sure, but this change is about using random port. I would like to revisit those tests to migrate to proper pytest parametrization later.

Reply:
You don't need rewriting the test to change asserts, this is not all our nothing to me. And about "this is not about it" argument - one can apply boyscout rule here) It's sure up to you.

Reply:
@Suor I usually do apply it, and I definitely don't want to leave problems for "the next one", but that test is messed up along with other ExternalRepro tests and need some fixup and there is absolutely no point in just changing self.assert -> assert without migrating from TestCase to pytest the test itself. It doesn't provide any benefit at all.

Reply:
It provides readability. And you know, readability counts)

Reply:
@Suor having regular asserts in TestCase test doesn't provide readability, because it is not what you expect there to see and code no longer becomes uniformed.